tests/mbis/PhOH.dat

```diff
@@ -0,0 +1,110 @@
+#! MBIS calculation on H2O
+
+charges_ref = psi4.Matrix.from_list([
```

Comment:
PhOH.dat and .out is a separate test, yes? Tests need to be one per directory and no variation in file names. So probably your original test directory becoming mbis1 and then a separate directory for the this phenol as mbis2 is a good plan. (Remember to change the `#!` line at the top -- this gets slurped up into the docs.)